v3/src/components/codap-component.tsx

```diff
@@ -14,17 +14,30 @@ export interface IProps extends ITileBaseProps {
   Component: React.ComponentType<ITileBaseProps>;
   tileEltClass: string;
   onCloseTile: (tileId: string) => void
+  onBottomRightPointerDown: (e: React.PointerEvent) => void
+  onBottomLeftPointerDown: (e: React.PointerEvent) => void
+  onRightPointerDown: (e: React.PointerEvent) => void
+  onBottomPointerDown: (e: React.PointerEvent) => void
+  onLeftPointerDown: (e: React.PointerEvent) => void
 }
 
-export const CodapComponent = observer(({ tile, TitleBar, Component, tileEltClass, onCloseTile }: IProps) => {
+export const CodapComponent =
+    observer(({ tile, TitleBar, Component, tileEltClass, onCloseTile, onBottomRightPointerDown, onBottomLeftPointerDown,
+      onRightPointerDown, onBottomPointerDown, onLeftPointerDown }: IProps) => {
   const dataset = gDataBroker?.selectedDataSet || gDataBroker?.last
 
   return (
     <DataSetContext.Provider value={dataset}>
       <div className={`codap-component ${tileEltClass}`}>
         <TitleBar tile={tile} onCloseTile={onCloseTile}/>
         <Component tile={tile} />
-        <ResizeHandle className="component-resize-handle"/>
+        <div className="codap-component-border right" onPointerDown={onRightPointerDown}/>
+        <div className="codap-component-border bottom" onPointerDown={onBottomPointerDown}/>
+        <div className="codap-component-border left" onPointerDown={onLeftPointerDown}/>
+        <div className="codap-component-corner bottom-left" onPointerDown={onBottomLeftPointerDown}/>
+        <div className="codap-component-corner bottom-right" onPointerDown={onBottomRightPointerDown}>
+          <ResizeHandle className="component-resize-handle"/>
+        </div>
```

Comment:
Make these conditional on whether the handler was provided -- if we don't have a handler then we don't need the DOM element. For mosaic only a subset of them will be necessary.